src/docs/tutorials/set-up-a-tailscale-subnet-router.md

```diff
@@ -0,0 +1,212 @@
+
+Deploying Tailscale as a subnet router into our project means that we can access the `railway.internal` private domains from any device connected to our tailnet.
+
+This tutorial aims to provide a simple step-by-step guide on setting up everything needed so that we can access the private domains of our services.
+
+**Objectives**
+
+In this tutorial, you'll learn how to do the following: -
+
+- Generate an Auth Key.
+- Set up split DNS.
+- Deploy the Tailscale Subnet Router template.
+- Approve the private network subnet.
+- (Bonus) Connect to Postgres locally via the private domain.
+
+**Prerequisites**
+
+If you're new to private networking, we strongly recommend checking out our private networking [guide](/guides/private-networking) and [reference](/reference/private-networking) page for a quick intro.
+
+**In Railway -**
+
+- Have all the services you plan on connecting to via the tailnet, listening on IPv6.
+
+    This is necessary because the Tailscale tunnel will communicate with your services over Railway's IPv6-only private network.
+
+    All database services already do this but for information on configuring your service to listen on IPv6, see [here](/guides/private-networking#listen-on-ipv6).
+
+**In Tailscale -**
+
+- Have an account.
+
+    You can sign up <a href="https://login.tailscale.com/start" target="_blank">here</a> - For what this template achieves you do not need a paid plan.
+
+- Have the Tailscale app installed on your computer.
+
+    You can find the downloads for your OS <a href="https://tailscale.com/download" target="_blank">here</a>.
+
+## 1. Getting an Auth Key
+
+The Auth key will authenticate the Tailscale machine that we'll deploy into our Railway project in a later step.
+
+- Head over to the [Keys](https://login.tailscale.com/admin/settings/keys) page located within the settings menu on the Tailscale dashboard.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1724349121/docs/tutorials/tailscale-subnet-router/keys_page_vohahp.png"
+alt="screenshot of the tailscale settings page"
+layout="intrinsic"
+width={1261} height={772} quality={100} />
+
+- Click **Generate auth key**.
+
+    Put in a description and leave all other settings as the default.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1724349121/docs/tutorials/tailscale-subnet-router/generate_auth_key_oxqr8m.png"
+alt="screenshot of the generate auth key modal in tailscale"
+layout="intrinsic"
+width={602} height={855} quality={100} />
+
+- Click **Generate key**.
+
+    Tailscale will now show you the newly generated auth key, **be sure to copy it down**.
+
+- Click **Done**.
+
+## 2. Configure Split DNS
+
+Without properly configuring our nameserver on Tailscale, we wouldn't be able to perform local DNS lookups for our private domains.
```

Comment:
Reads a bit awkward- like we are padding the user from a negative externality. In documentation writing (at least I was taught) - we should affirm the reason why we do something before we warn them of the bad outcome.

Like: "Next, we need to make the service discoverable to our computer. If we dont...  we wouldn't be able to perform local DNS lookups for our private domains and can't connect."